build.sh

```diff
@@ -65,33 +65,30 @@ initHostDistroRid()
 {
     __HostDistroRid=""
     if [ "$__HostOS" == "Linux" ]; then
-        if [ -e /etc/os-release ]; then
-            source /etc/os-release
-            if [[ $ID == "rhel" ]]; then
-                # remove the last version digit
-                VERSION_ID=${VERSION_ID%.*}
-            fi
-            __HostDistroRid="$ID.$VERSION_ID-$__HostArch"
-            if [[ $ID == "alpine" ]]; then
-                __HostDistroRid="linux-musl-$__HostArch"
-            fi
-        elif [ -e /etc/redhat-release ]; then
+        if [ -e /etc/redhat-release ]; then
```

Comment:
I'm just curious - why have you flipped the order of checks for /etc/redhat-release and /etc/os-release?

Reply:
On RHEL 7 the both files are present - so the flipping is to do RHEL specific check first to have consistency across different versions of RedHat/CentOS. Do you see any disadvantages of doing it this way?

Reply:
It seems that we were actually lucky that on CentOS 7 we were using the /etc/os-release. While for CentOS 6 we use rhel.6 RID, it is only because we officially support only RHEL 6 - but we build the binaries on centos 6. For version 7 though, we should use centos.7 / rhel.7. 
So please keep the original way of getting info from the /etc/os-release first. And please also remove the check for centos under the /etc/redhat-release `if` branch